stdlib/test/base64/test_base64.mojo

```diff
@@ -12,11 +12,12 @@
 # ===----------------------------------------------------------------------=== #
 # RUN: %mojo %s
 
-from base64 import b64encode
+from base64 import b64encode, b64decode
 from testing import assert_equal
 
 
 def test_b64encode():
+    print("== test_b64encode")
```

Comment:
**Suggestion** Drop the `print` here and in the decode function below.  The `print(...)` is only needed in `FileCheck`-style tests so the tool has something to latch onto for its regex-matching internally.